projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -34,6 +34,13 @@ class Admin_Menu {
 	 */
 	protected $domain;
 
+	/**
+	 * The customizer default link.
+	 *
+	 * @var string
+	 */
+	protected $customize_slug;
```

Comment:
If you set the default here, it doesn't need to be set in the constructor
```suggestion
	protected $customize_slug = 'customize.php';
```